exercises/custom-set/custom_set_test.rb

```diff
@@ -3,114 +3,303 @@
-    assert_equal CustomSet.new([1, 2, 3]), CustomSet.new([3, 2, 1]).delete(4)
-    assert_equal CustomSet.new([1, 2, 3]), CustomSet.new([3, 2, 1]).delete(2.0)
-    assert_equal CustomSet.new([1, 3]), CustomSet.new([3, 2.0, 1]).delete(2.0)
+    set1 = CustomSet.new [1, 2]
+    set2 = CustomSet.new [2, 3]
+    refute set1.disjoint? set2
   end
 
-  def test_difference
+  def test_sets_are_disjoint_if_they_share_no_elements
     skip
-    assert_equal CustomSet.new([1, 3]),
-                 CustomSet.new([1, 2, 3]).difference(CustomSet.new([2, 4]))
+    set1 = CustomSet.new [1, 2]
+    set2 = CustomSet.new [3, 4]
+    assert set1.disjoint? set2
+  end
 
-    assert_equal CustomSet.new([1, 2.0, 3]),
-                 CustomSet.new([1, 2.0, 3]).difference(CustomSet.new([2, 4]))
+  def test_empty_sets_are_equal
+    skip
+    set1 = CustomSet.new []
+    set2 = CustomSet.new []
+    assert_equal set1, set2
   end
 
-  def test_disjoint?
+  def test_empty_set_is_not_equal_to_non_empty_set
     skip
-    assert CustomSet.new([1, 2]).disjoint?(CustomSet.new([3, 4]))
-    refute CustomSet.new([1, 2]).disjoint?(CustomSet.new([2, 3]))
-    assert CustomSet.new([1.0, 2.0]).disjoint?(CustomSet.new([2, 3]))
-    assert CustomSet.new.disjoint?(CustomSet.new)
+    set1 = CustomSet.new []
+    set2 = CustomSet.new [1, 2, 3]
+    refute_equal set1, set2
   end
 
-  def test_empty
+  def test_non_empty_set_is_not_equal_to_empty_set
     skip
-    assert_equal CustomSet.new, CustomSet.new([1, 2]).empty
-    assert_equal CustomSet.new, CustomSet.new.empty
+    set1 = CustomSet.new [1, 2, 3]
+    set2 = CustomSet.new []
+    refute_equal set1, set2
   end
 
-  # rubocop:disable Metrics/LineLength
-  def test_intersection
+  def test_sets_with_the_same_elements_are_equal
     skip
-    assert_equal CustomSet.new([:a, :c]),
-                 CustomSet.new([:a, :b, :c]).intersection(CustomSet.new([:a, :c, :d]))
+    set1 = CustomSet.new [1, 2]
+    set2 = CustomSet.new [2, 1]
+    assert_equal set1, set2
+  end
 
-    assert_equal CustomSet.new([3]),
-                 CustomSet.new([1, 2, 3]).intersection(CustomSet.new([1.0, 2.0, 3]))
+  def test_sets_with_different_elements_are_not_equal
+    skip
+    set1 = CustomSet.new [1, 2, 3]
+    set2 = CustomSet.new [1, 2, 4]
+    refute_equal set1, set2
   end
 
-  def test_member?
+  def test_add_to_empty_set
     skip
-    assert CustomSet.new([1, 2, 3]).member?(2)
-    assert CustomSet.new(1..3).member?(2)
-    refute CustomSet.new(1..3).member?(2.0)
-    refute CustomSet.new(1..3).member?(4)
+    set = CustomSet.new []
+    element = 3
+    expected = CustomSet.new [3]
+    assert_equal expected, set.put(element)
   end
 
-  def test_put
+  def test_add_to_non_empty_set
     skip
-    assert_equal CustomSet.new([1, 2, 3, 4]),
-                 CustomSet.new([1, 2, 4]).put(3)
+    set = CustomSet.new [1, 2, 4]
+    element = 3
+    expected = CustomSet.new [1, 2, 3, 4]
+    assert_equal expected, set.put(element)
+  end
```

Comment:
Something about this seems a little awkward to me. As I read each line I don't understand why its important. Perhaps something like?
```
set1 = CustomSet.new [1, 2, 3, 4]
set2 = CustomSet.new [1, 2, 4]
assert_equal set1, set2.put(3)
```